cypress/e2e/download/download-file.cy.js

```diff
@@ -0,0 +1,23 @@
+// Description of task: User clicks on the 'download file' button
+// Expected results: File is verified as downloaded
+
+describe("download-file", () => {
```

Comment:
The testing functionality in this and the following test files in this commit (each describe + it function definitions) are the first implementation of UI-based testing for the download page before later test organization (describe, context, and multiple it functions per context) was established.